network-api/networkapi/management/commands/flush_models.py

```diff
@@ -3,6 +3,7 @@
 # Models
 from networkapi.highlights.models import Highlight
 from networkapi.landingpage.models import LandingPage, Signup
+from networkapi.campaign.models import CampaignPage
```

Comment:
```
    from networkapi.campaign.models import CampaignPage
ImportError: cannot import name 'CampaignPage'

```

Reply:
This is what I get for trying to multitask